lesson13/js/index.js

```diff
@@ -0,0 +1,81 @@
+const wrapper = document.getElementById('js-wrapper');
+const modalBtn = document.getElementById('js-modal-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modal = document.getElementById('js-modal');
+
+
+function addLoading() {
+  const loading = document.createElement('img');
+  loading.src = "./img/loading-circle.gif";
+  loading.id = "loading";
+  wrapper.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+async function fetchErrorHandling(response) {
+  if (!response.ok) {
+    throw new Error("サーバーエラーが発生しました");
+  } else {
+    return await response.json();
+  }
+}
+
+async function fetchData() {
+  const response = await fetch("https://jsondata.okiba.me/v1/json/fR9k5211007080400");
+  const json = await fetchErrorHandling(response);
+  return json;
+}
+
+function createLists(values) {
+  const frag = document.createDocumentFragment();
+  const ul = document.getElementById('js-ul');
+
+  values.data.forEach(value => {
+    const li = document.createElement('li');
+    const anchor = document.createElement('a');
+    const img = document.createElement('img');
+
+    anchor.href = `/${value.to}`;
```

Comment:
The href is set to undefined.
In this code, the error handling of the json data acquisition is done, but the error handling of the acquired data itself is not done.

Reply:
Thank you very much.
The retrieved data was not reflected because the attribute value was incorrectly specified.
We have fixed it. e37a7d6